samples/csharp/MicrosoftGraphSDKBeta/GraphException.cs

```diff
@@ -12,12 +12,16 @@ namespace MicrosoftGraphSDK
     internal class GraphException : Exception
```

Comment:
for this scenario, it would be better if this type was `public sealed` (since it's being thrown from `public` methods) so callers can `catch` it easily.

Reply:
also--is there a way to share code between the Beta folder and the v1.0 folder? It'd be easier to maintain if we didn't have multiple copies of these "common" types.